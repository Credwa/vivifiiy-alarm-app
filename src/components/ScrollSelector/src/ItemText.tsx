import React, { FC } from 'react';
import { Text, TextStyle } from 'react-native';
import VivText from '@/components/VivText';
import Colors from '@/constants/Colors';

interface ITextProps {
  style?: TextStyle;
  fontName: any;
  color: string;
}
const ItemText: FC<ITextProps> = ({ fontName, color, children, style }) => (
  <VivText fontName={fontName} color={color}>
    {children}
  </VivText>
);

export default ItemText;
