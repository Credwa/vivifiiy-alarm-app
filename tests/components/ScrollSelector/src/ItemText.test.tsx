import React from 'react';
import renderer from 'react-test-renderer';
import ItemText from '@/components/ScrollSelector/src/ItemText';

describe('ScrollSelector/src/ <ItemText />', () => {
  it(`renders correctly`, () => {
    const tree = renderer.create(<ItemText fontName={''} color={'red'} />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
