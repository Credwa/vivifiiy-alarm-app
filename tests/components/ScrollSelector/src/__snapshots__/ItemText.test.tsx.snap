// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ScrollSelector/src/ <ItemText /> renders correctly 1`] = `
<Text
  style={
    Array [
      Object {
        "color": "#869DB2",
      },
      Object {
        "fontFamily": "Nunito_400Regular",
        "fontSize": 17,
      },
      Object {
        "color": "red",
      },
      undefined,
    ]
  }
/>
`;
